src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -484,6 +521,7 @@ winrt::guid Profile::GetGuidOrGenerateForJson(const Json::Value& json) noexcept
 Json::Value Profile::ToJson() const
 {
     Json::Value json{ Json::ValueType::objectValue };
+    auto defaultAppearanceImpl = winrt::get_self<implementation::AppearanceConfig>(_DefaultAppearance);

```

Comment:
Again, could we save some work by initializing `json` to `defaultAppearanceImpl->ToJson()`? Then `Profile.cpp` shouldn't need any of the json keys for the appearance settings at all